contracts/KyberNetwork.sol

```diff
@@ -238,21 +248,21 @@ contract KyberNetwork is Withdrawable, Utils {
     }
     /* solhint-enable code-complexity */
 
-    function getExpectedRate(ERC20 source, ERC20 dest, uint srcQuantity)
+    function getExpectedRate(ERC20 source, ERC20 dest, uint sourceQty)
         public view
         returns (uint expectedRate, uint slippageRate)
     {
-        require(expectedRateContract != address(0));
-        return expectedRateContract.getExpectedRate(source, dest, srcQuantity);
+        require(enabled);
```

Comment:
it shouldn't check for enable. it should check for address != 0

Reply:
but all addresses were checked upon init.
if we had a few addresses here it would be enough to check enable.

this was his remark. put all address validation in one place. preffered is constructor. but here we are limited so here its the setParams

Reply:
I want to call it even before enable = true

Reply:
OK